debian/control

```diff
@@ -0,0 +1,132 @@
+Source: bm
+Section: net
+Priority: optional
+Maintainer: Thomas Dreibholz <dreibh@simula.no>
+Build-Depends: automake,
+               cmake,
+               curl,
+               debhelper (>= 12),
+               dh-python,
+               git,
+               libboost-dev,
+               libboost-filesystem-dev,
+               libboost-program-options-dev,
+               libboost-system-dev,
+               libboost-thread-dev,
+               libgmp-dev,
+               libgrpc++-dev,
+               libgrpc-dev,
+               libjudy-dev,
+               libnanomsg-dev,
+               libpcap-dev,
+               libprotobuf-dev,
+               libprotoc-dev,
+               libssl-dev,
+               libthrift-dev (>= 0.13.0),
+               libtool,
+               pkg-config,
+               protobuf-compiler,
+               protobuf-compiler-grpc,
+               python3-all,
+               python3-six,
+               python3-thrift,
+               thrift-compiler
+Standards-Version: 4.5.1.0
+Rules-Requires-Root: no
+Homepage: https://github.com/simula/behavioral-model
+Vcs-Git: https://github.com/simula/behavioral-model
+Vcs-Browser: https://github.com/simula/behavioral-model/tree/p4-packaging
+
+Package: p4lang-bmv2-bin
+Architecture: amd64 arm64
+Depends: p4lang-bmv2-libs (= ${binary:Version}),
+         python3-bmv2 (= ${binary:Version}),
+         ${misc:Depends},
+         ${python3:Depends},
+         ${shlibs:Depends}
+Recommends: hipercontracer,
+            netperfmeter,
+            rsplib-tools,
+            subnetcalc,
+            tcpdump,
+            tsctp,
+            tshark
+Description: p4lang behavioral-model (binaries)
```

Comment:
I believe this list of packages is only "recommended", not "required", but hipercontracer for example does not even exist in Ubuntu 18.04 package system.  Some of the others I've never heard of before either.  Would it be better to recommend some of them in a README or example of other interesting tools to help debug things, rather than auto-recommended when installing this package?

Reply:
I just removed the "Recommended" part.